Browser_IDE/stylesheet.css

```diff
@@ -303,11 +348,34 @@ li.CodeMirror-hint-active {
     color: white;
 }
 
+#fileViewContainer,
+#codeViewContainer,
+#runtimeContainer {
+  padding: 10px;
+}
 
 
+/* New changes for scrollbar in FILES explorer section */
 
+.sk-column#fileViewContainer {
+  max-height: calc(100vh - 3em); 
+  overflow: auto; 
+}
+
+#fileViewContainer .sk-contents-focusable {
+  height: calc(100% - 50px); 
+  overflow-y: auto;
+  overflow-x: auto;  
+  width: 100%; 
+  max-width: 100%; 
+}
 
 
+#fileView > * {
```

Comment:
Not sure about this one - while I like having the horizontal scrollbar appear when a filename is long, I don't like the awkward spacing when the files are named at reasonable lengths (like in the image Jess showed). I think for now it's better to remove it, but if you can find a way to fix the awkward spacing then that'd be fine too  😃